types/index.d.ts

```diff
@@ -894,6 +894,22 @@ declare namespace math {
       threshold?: number
     ): MathArray
 
+    /* Finds the roots of a polynomial of degree three or less. Coefficients are given constant first
+     * followed by linear and higher powers in order; coefficients beyond the degree of the polynomial
+     * need not be specified.
+     * @param {number|Complex} constantCoeff
+     * @param {number|Complex} linearCoeff
+     * @param {number|Complex} quadraticCoeff
+     * @param {number|Complex} cubicCoeff
+     * @returns {Array<number|Complex>} array of roots of specified polynomial
+     */
+    polynomialRoot(
```

Comment:
Is the function usable in chained method calls? I guess that doesn't make sense? At least there is no TypeScript definition for the chained method call right now.

Reply:
Nope, no chained method; don't see how it makes any sense to split the coefficients of a polynomial with the constant term in the chain and the linear and up in the polynomialRoot call...